### PR TITLE
[stable/kibana] Add upgrading section to README

### PR DESCRIPTION
**Pull request by naseemkullah**
Signed-off-by: Naseem <naseemkullah@gmail.com>

#### What this PR does / why we need it:

As per @desaintmartin's comment https://github.com/helm/charts/pull/13355#issuecomment-491818888

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Special notes for your reviewer:

cc @monotek 

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] title of the PR contains starts with chart name e.g. `[stable/chart]`

### CODE REVIEW

**Review by monotek — APPROVED** [review]
/lgtm

File: stable/kibana/README.md
Comment (desaintmartin) [inline stable/kibana/README.md]:
```suggestion
### To 2.3.0
```

**Comment by desaintmartin** [comment]
I think we commented at the same time. :)

Open question: should this PR bump a major version?

**Comment by naseemkullah** [comment]
> I think we commented at the same time. :)
> 
> Open question: should this PR bump a major version?

I'm not sure honestly. I suppose so since it could be a breaking change for people who explicitly set the value of `elasticsearch.url`.

Another option is to leave both uncommented (`elasticsearch.url` and `elasticsearch.hosts`), then the minor bump would be sufficient since it would be backwards compatible... Let me know what you think is best.

**Comment by naseemkullah** [comment]
> I think we commented at the same time. :)
> 
> Open question: should this PR bump a major version?

I'm not sure honestly.

If people are explicitly setting their value for `elasticsearch.url` it will continue to be explicitly set and thus a backwards compatible change

Only way it would break is  if people are pinning to version kibana < 6.6 and not explicitly setting elasticsearch.url, so i suppose it is a breaking change.

Your call!

**Comment by desaintmartin** [comment]
From my understanding, without configuration change when upgrading to 2.3.0, both `elasticsearch.url` and `elasticsearch.host` will be set. Which one will be used?

**Comment by monotek** [comment]
> From my understanding, without configuration change when upgrading to 2.3.0, both `elasticsearch.url` and `elasticsearch.host` will be set. Which one will be used?

I think it depends on the used kibana version.

**Comment by naseemkullah** [comment]
> > From my understanding, without configuration change when upgrading to 2.3.0, both `elasticsearch.url` and `elasticsearch.host` will be set. Which one will be used?
> 
> I think it depends on the used kibana version.

Prior to 6.6, only .url will be used. Not entirely sure for 6.6+ ... in any case if things break one can refer to the hereby Upgrading section  and configure their values files with just .host :)

**Comment by naseemkullah** [comment]
Personally I was unable to run 6.6 without the `elasticsearch.host` setting, has anyone succeeded in running 6.6+ with only `elasticsearch.url` setting?

**Comment by compleatang** [comment]
I see no harm in bumping a major version here. Mostly as a signal to users that something significant has changed. Good to keep backwards compatibility imv for a few more kibana versions. But bumping a major version on the chart due to this relatively significant change (albeit largely by the consuming runtime rather than the chart itself) is proper. I don't hold this opinion strongly tho and I can also see arguments against.

**Comment by naseemkullah** [comment]
Good points @compleatang 
Major version bumped!
Let's get this in :)

**Review by monotek — APPROVED** [review]
/lgtm

**Comment by desaintmartin** [comment]
Thanks!

File: stable/kibana/README.md
Comment (desaintmartin) [inline stable/kibana/README.md]: Uh, this was not 2.3.0. :)

**Pull request merged**
